--- "tmp/01.sentences/Su\305\241il_prosodie-List1-export-20260220.txt"	2026-02-20 17:51:53.384102941 +0100
+++ "tmp/01.sentences/Su\305\241il_prosodie-List1-export-20260220.fixed.txt"	2026-02-20 17:50:18.611970189 +0100
@@ -27,12 +27,12 @@
 Tyto a jiné tu vypuštěné stopy znáti třeba kde se řeč vede o veršech řeckých a latinských; leč pro českou prosodii postačí to co o dvou-, tři a čtveroslabičných stopách napovědíno.
 Pravidlem kterým se veršové nesou a chod jímž se sestavují, sluje metrum či rozměr.
 Pravidlem kterým se veršové nesou a chod jímž se sestavují, sluje metrum či rozměr.
-Tudy se říká že jambické, trochaické a ana- péstické verše dipodicky či dvoustopně, daktylické monopodicky či jednostopně se rozměřují.
+Tudy se říká že jambické, trochaické a anapéstické verše dipodicky či dvoustopně, daktylické monopodicky či jednostopně se rozměřují.
 Tudy se říká že jambické, trochaické a anapéstické verše dipodicky či dvoustopně, daktylické monopodicky či jednostopně se rozměřují.
 Verš, jehož poslední stopa neúplna jest, sluje nedoměrným či katalektickým; verš, v němž syllaba nadbývá, sluje nadměrným či hyperkatalektickým, an verš úplný a dokonalý doměrným či skonalým, akatalektickým se nazývá.
 Již výše praveno že stopa slovná nemá se scházeti se stopou veršovou a že toho libozvuk a hudebnost verše vymáhá, aby stopa slovná se stopou veršovou se vespolek přerývaly a slovo chod stopy přetrhovalo.
 Již výše praveno že stopa slovná nemá se scházeti se stopou veršovou a že toho libozvuk a hudebnost verše vymáhá, aby stopa slovná se stopou veršovou se vespolek přerývaly a slovo chod stopy přetrhovalo.
-Dále rozeznávají se přerývky stálé a změnné, jejichž jména sama je vysvětlují; níže při roz- měrech zračitě se vyskytnou.
+Dále rozeznávají se přerývky stálé a změnné, jejichž jména sama je vysvětlují; níže při rozměrech zračitě se vyskytnou.
 Dále rozeznávají se přerývky stálé a změnné, jejichž jména sama je vysvětlují; níže při rozměrech zračitě se vyskytnou.
 Od přerývky té stopné rozlikuje se přerývka slovná jenž záleží v tom, aby slovo totéž s jedné stopy do druhé přecházelo; takový přechod slova se stopy do stopy sluje přerývka slovná.
 Již sice přerývka předešlá, stopová, k hudebnosti působí a jazykové někteří n. p. anglický na ní přečasto přestávati musejí; ale u klassických národů a v naší hudebné řeči nelze se bez ní obejíti.
@@ -98,7 +98,7 @@
 V básních přízvučných (jinak jest ve verších časoměrných) pro čtení a deklamování, nikoliv pak pro zpěv psaných, nesmí se rýmovati slovo dvojslabičné s tréslabičným, jako: voda - - náhoda, což vysvítá z toho, co již na hoře o pří- zvuku praveno bylo.
 Rýmův s úzkými samohláskami' e, i, zvláště mají-li dvě i v sobě, v básně ch/ vůbec pro nelibozvučnost, zvláště pak v písních pro nezpěvnost velmi pořídku upotřebiti se má; k. p. choditi diti - roditi - shoditi -- ploditi
 U Španělů se assonance zhusta užívá; ale jest assonance vezdy jen nedochůdčetem, a ač v pisněch národních někde přicházejíc nesmí sobě osvojiti místo v básni vzdělané.
-Totéž slovo dvakrát opakované není rýmem; k. p. chodí - -- nepochodí;  lečby totéž slovo v každém verši v jiném smyslu se bralo;
+Totéž slovo dvakrát opakované není rýmem; k. p. chodí - -- nepochodí; lečby totéž slovo v každém verši v jiném smyslu se bralo;
 Podobně se má s dvojím I, totiž s tvrdým a s měkkým I.
 Nebo ačkoliv polovice Moravy a celé Slovensko a Slezko toho jak i předešlého rozdílu dobře vědomo jest, přece v jazyku spisovném již dávno rozdílu toho se nešetří, a tudyž volno rýmovati tvrdé I s měkkým 1; k. p. králi stály; voly v poli; měly pěli.
 Rýmy ty se naznačují: a a. Rýmy střídavé jsou, když se verš první s třetím, a druhý se čtvrtým rýmuje.
